addon/components/o-s-s/progress-bar.hbs

```diff
@@ -0,0 +1,19 @@
+<div class={{concat 'oss-progress-bar ' this.computedSize}} ...attributes>
+  {{#if @label}}
+    <span class='oss-progress-bar__label'>
+      {{@label}}
+    </span>
+  {{/if}}
+  <div class='oss-progress-bar__outer'>
+    <div
+         class={{concat 'oss-progress-bar__inner ' this.computedStyles}}
+         role='progressbar' aria-valuenow={{@value}}
+         aria-valuemin='0' aria-valuemax='100' style={{progressBarWidthStyle}}>
+    </div>
+  </div>
+  {{#if @displayValue}}
+    <span class='oss-progress-bar__value'>
+      {{@value}}%
+    </span>
+  {{/if}}
+</div>
```

Comment:
Please use double quotes in template files

single quotes are okay inside double quotes, eg:
`class="button {{if @param 'other-class'}}'"`

Reply:
Fixed in lastest commit